aws/network-primitives/instances.tf

```diff
@@ -0,0 +1,297 @@
+resource "aws_instance" "bootstrap_node" {
+  count             = length(var.aws_region) * var.instance_count
+  ami               = data.aws_ami.ubuntu_amd64.image_id
+  instance_type     = var.instance_type
+  subnet_id         = element(aws_subnet.public_subnets.*.id, count.index)
+  availability_zone = element(var.azs, count.index)
+  # Security Group
+  vpc_security_group_ids = ["${aws_security_group.gemini_sg.id}"]
+  # the Public SSH key
+  key_name                    = var.aws_key_name
+  associate_public_ip_address = true
+  ebs_optimized = true
+  ebs_block_device {
+    device_name = "/dev/sda1"
+    volume_size = 100
```

Comment:
These disk sizes look severely underestimated. Are you sure they are sufficient?

Reply:
actually, I will set these in configs.

Reply:
Well, you did, but as far as I see you have used those 50G sized that I complained about in the other comment. So we have kind of made a full circle now. Or am I missing something and those values come from somewhere else?

Reply:
Actually,  the comment is outdated, not sure why you're still seeing that.

Reply:
This comment is outdated, but the issue is not, which is why I'm continuing this conversation. I see several places and none of them have the correct disk size. So to the best of my understanding we still have an issue even though code has changed since I wrote the first comment here.